src/main/resources/reference.conf

```diff
@@ -31,6 +31,14 @@ com.socrata.query-coordinator {
     address = "localhost"
   }
 
+  liveness-check {
+    ## Bind to address of the specific hostname or IP if specified, otherwise use wildcard. This should be set on
+    ##  systems with multiple interfaces on the same network or you may risk sending responses from the wrong IP.
+    #address = "localhost"
+    ## Listen on port if specified, otherwise use ephemeral port. 
+    #port = 6031
```

Comment:
Not by default, that is just an example.  In the general case we want an
ephemeral port, it is only for the docker config we want it fixed so that
is only set in the docker/query-coordinator.conf.j2.

On Mon, Feb 23, 2015 at 5:39 PM, John Krah <notifications@github.com> wrote:

> In src/main/resources/reference.conf
> <https://github.com/socrata/query-coordinator/pull/28#discussion_r25221677>
> :
>
> > @@ -31,6 +31,14 @@ com.socrata.query-coordinator {
> >      address = "localhost"
> >    }
> >
> > +  liveness-check {
> > +    ## Bind to address of the specific hostname or IP if specified, otherwise use wildcard. This should be set on
> > +    ##  systems with multiple interfaces on the same network or you may risk sending responses from the wrong IP.
> > +    #address = "localhost"
> > +    ## Listen on port if specified, otherwise use ephemeral port.
> > +    #port = 6031
>
> did you intend to bind the specific port 6031 ? it is exposed in
> dockerfile
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/socrata/query-coordinator/pull/28/files#r25221677>.
>